app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -113,8 +113,8 @@ def stringify_attr(attribute)
 
   def ial2_data
     @ial2_data ||= begin
-      if ial2_session? || ialmax_session?
-        out_of_band_session_accessor.load_pii
+      if (ial2_session? || ialmax_session?) && active_profile.present?
```

Comment:
Do we want a method that expresses `ial2_session? || ialmax_session?`, maybe `profile_requested?` or something like that?

Reply:
I think within the context of the user info controller these make sense as different request contexts where PII may be read. I think that reducing them to a single method may add some indirection.